MAINTAINERS.adoc

```diff
@@ -118,9 +127,15 @@ If you think some information is outdated, either provide a pull request or send
 |Kubernetes Client
 |https://github.com/geoand[Georgios Andrianakis], https://github.com/gastaldi[George Gastaldi]
 
+|Liquibase
+|https://github.com/geoand[Georgios Andrianakis]
```

Comment:
I think we should also add https://github.com/andrejpetras .

Reply:
Good idea, done